### PR TITLE
(#513) Modify delighter URLs not to point to specific generated sprite

### PR DESCRIPTION
**Pull request by alerosad**
(#Closes 513)
- updated icons to no longer be retrieved from https://www.cancer.gov/PublishedContent/images/images/design-elements/sprites/svg-sprite.__v15577802.svg
- icons are now base64 encoded, then decoded

### CODE REVIEW

**Comment by alinai** [comment]
@alerosad, verified that each delighter icon is individually encoded for whatever is needed, rather than the whole sheet being encoded using the online decoder. Verified the Basic Search page, Advanced Search page, Search Results page, Trial Description page, No Trial Found page, Invalid Search Criteria page. Thanks @alerosad for providing the test step for this.

@sarinapadilla , Noticed that when we hover on the

tag under the delighter we used to get underline in Prod. We are not seeing the underline on hovering the

tag under the delighter on https://react-app-dev.cancer.gov/clinical-trials-search-app/pr-518/. @alerosad is checking this.

**Comment by SupriyaSaxena0605** [comment]
Thanks @alerosad for providing the test steps. Verified image source for each delighter on react app with respect to the same on prod using the decoder. Tested the same for the delighters on CTS search forms, search results page, trial description page, No trials found page and Invalid search criteria page.

**Comment by alerosad** [comment]
the p underline tag issue you mention is apparent on the feature branch as well, seems to be an existing issue, unrelated to this ticket

**Comment by alinai** [comment]
> the p underline tag issue you mention is apparent on the feature branch as well, seems to be an existing issue, unrelated to this ticket

Thanks @alerosad for confirming!

**Comment by SupriyaSaxena0605** [comment]
Thanks for confirming @alerosad

**Review by adriancofie — APPROVED** [review]

File: src/views/TrialDescriptionPage/TrialDescriptionPage.scss [hunk 1]
```diff
@@ -1,306 +1,310 @@
...
-  }
+	.icon {
+		.icon-print {
+			background-image: url('data:image/svg+xml;base64, PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9Ii0xMSAtMTEgNDUgNDEiIHdpZHRoPSI0NSIgaGVpZ2h0PSI0MSIgaWQ9InByaW50ZXItd2hpdGUiIHk9IjMyOTYiPjxwYXRoIGQ9Ik0xOS4wMyAxNC4yNUgyM3YtOS41aC0zLjk3VjBIMy45OHY0Ljc1SDB2OS41aDMuOTdWMTloMTUuMDd2LTQuNzV6TTE4LjI0Ljc5djMuOTZINC43N1YuNzltMCA4LjA1aDEzLjQ5djkuMzdINC43N1Y4Ljg0eiIgZmlsbD0iI2ZmZiIvPjxwYXRoIGNsYXNzPSJjb3N0MCIgZmlsbD0iI2ZmZiIgZD0iTTUuNTYgMTMuNDZoMTEuOXYuNzlINS41NnptMCAyLjM4aDExLjl2Ljc5SDUuNTZ6bTAtNC43NmgxMS45di43OUg1LjU2eiIvPjwvc3ZnPgo=');
+		}
+		.icon-email {
+			background-image: url('data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9Ii0xMSAtMTEgNDUgMzYiIHdpZHRoPSI0NSIgaGVpZ2h0PSIzNiIgaWQ9Im1haWwtZW52ZWxvcGUtY2xvc2VkLXdoaXRlIiB5PSIyNjY5Ij48cGF0aCBkPSJNMjEuNSAwbC0xMCA4LjdMMS41IDBoMjB6TS4zMSAxMy43M2MtLjE5LS4xOS0uMzEtLjQ1LS4zMS0uNzRWMS4wMUMwIC43Mi4xMi40Ni4zMi4yOEw4LjA1IDcgLjMxIDEzLjczek0xLjUgMTRsNy4zLTYuMzVMMTEuNSAxMGwyLjctMi4zNUwyMS41IDE0aC0yMHpNMjMgMTIuOTljMCAuMjktLjEyLjU1LS4zMi43M0wxNC45NSA3IDIyLjY5LjI3Yy4xOS4xOS4zMS40NS4zMS43NHYxMS45OHoiIGZpbGw9IiNmZmYiLz48L3N2Zz4K');
+		}
+		display: inline-block;
```
Comment (arcepaul) [inline src/views/TrialDescriptionPage/TrialDescriptionPage.scss:120]: this block should be at the top

File: src/views/TrialDescriptionPage/TrialDescriptionPage.scss [hunk 2]
```diff
@@ -1,306 +1,310 @@
...
-    font-size: 14px;
-    font-weight: bold;
-    border: 0 none;
-    cursor: pointer;
-    @media screen and (min-width: 1025px) {
-      width: 47%;
-    }
-  }
+	.icon {
+		.icon-print {
```
Comment (arcepaul) [inline src/views/TrialDescriptionPage/TrialDescriptionPage.scss:114]: class="icon icon-print" so you need to prepend with `&`

```
.icon {
     // .icon rules here
     &.icon-print {}
     &.icon-email {}
}
```

**Review by arcepaul — APPROVED** [review]

**Pull request merged**